packages/iris-grid/src/sidebar/RollupRows.tsx

```diff
@@ -78,9 +78,11 @@ class RollupRows extends Component<RollupRowsProps, RollupRowsState> {
     isClone?: boolean;
     selectedCount?: number;
   }): ReactElement {
-    const text = item && item.name;
-    const badgeText = isClone ? `${selectedCount}` : undefined;
-    const className = isClone ? 'item-list-item-clone' : '';
+    const text = item != null ? item.name : undefined;
```

Comment:
I think you can just use optional chaining
```suggestion
    const text = item?.name;
```